qubes-i3status

```diff
@@ -80,7 +80,26 @@ status_qubes() {
 }
 
 status_disk() {
-    local disk=`df -h / | tail -n 1 | awk '{print $4}'`
+    local disk=''
+    local free=''
+    local size=''
+    local usage=''
+    read size usage <<< $(qvm-pool -i lvm | grep 'size\|usage' | sort | awk '{print $2}')
```

Comment:
I'd recommend adding `$(qubes-prefs default-pool)` instead of hardcoding `lvm` - this way it will also work on btrfs installation (or custom pools).

Reply:
Added commit, addressing that.